File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
  let arr = binomial.split(" ")
  let result = 0

  switch(arr[1]) {
    case "+":
      result = Number(arr[0]) + Number(arr[2])
      break
    case "-":
      result = Number(arr[0]) - Number(arr[2])
      break
    case "*":
      result = Number(arr[0]) * Number(arr[2])
      break
  } 

  return result
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
  public int solution(String binomial) {
    String[] arr = binomial.split(" ");
    int ans = 0;

    switch (arr[1]) {
      case "+":
        ans = Integer.parseInt(arr[0]) + Integer.parseInt(arr[2]);
        break;
      case "-":
        ans = Integer.parseInt(arr[0]) - Integer.parseInt(arr[2]);
        break;
      case "*": 
       ans = Integer.parseInt(arr[0]) * Integer.parseInt(arr[2]);
       break;
    }

    return ans;
  }
}